pynest/nest/lib/hl_api_info.py

```diff
@@ -55,23 +58,23 @@ def authors():
 
 
 @check_stack
-def helpdesk(browser="firefox"):
-    """Open the NEST helpdesk in the given browser.
+def helpdesk():
+    """Open the NEST helpdesk in browser.
 
-    The default browser is firefox.
-
-    Parameters
-    ----------
-    browser : str, optional
-        Name of the browser to use
+    The system default browser.
+    /helpdesk << /command (firefox) >> SetOptions in ~/.nestrc is now obsolete
     """
-
-    sr("/helpdesk << /command (%s) >> SetOptions" % browser)
-    sr("helpdesk")
+    url = os.path.join(os.environ['NEST_DOC_DIR'] + "/help", "helpindex.html")
+    if sys.platform[:3] == "win":
+        os.startfile(url)
+    if sys.platform[:3] == "dar":
+        webbrowser.get('safari').open_new(url)
```

Comment:
Why not `webbrowser.open_new(url)` also here? Ok, due to [this problem in OSX](http://www.andrewjaffe.net/blog/2017/05/python-bug-hunt.html) we need to ask for the browser explicitly under macOS are present (10.12.5). Worse, a failing `webbrowser.open_new(url)` does not even raise a Python exception but returns happily. So I think we need to keep this, but i would be good to document why we do this.